### ISSUE TITLE
Minor improvements to main screen UI

### ISSUE THREAD

**Issue by jurajhilje** [ivpn/android-app#85]
Remove "Connection details" header - It doesn't make any sense as connection details are above and below.

Remove latency dot from main window UI (not server selection window). It's possible that this could be confused with connection status which could lead to user thinking they are connected when they are not and it doesn't provide much useful information anyway. If I regularly connect to Germany and notice on the main Window UI that server latency has changed from green to yellow then what am I going to do? Change to another server which is green? If so then why not just use the 'fastest server' feature which will automatically do everything?

Add text below VPN protocol selection:
"We recommend the WireGuard protocol for its speed, security and connection reliability. Currently we only support Multi-hop connections using OpenVPN. For more information visit our [protocol comparison web page](https://www.ivpn.net/pptp-vs-ipsec-ikev2-vs-openvpn-vs-wireguard/)."

**Comment by gorkapernas** [comment]
Verified on version 2.7.0, Samsung Galaxy S10 Android 11, Huawei MediaTab Android 8.1, the changes have been implemented accordingly. See screenshot below for reference.

![text changes](https://user-images.githubusercontent.com/16647612/131342002-f933474d-19d4-457d-b21a-0d49a027f2a7.png)

**Comment by gorkapernas** [comment]
@AlexKorvin I just realised that the there is an inconsistency between the implementations on our different apps.
The requirements were not clear, but apparently the text should have been implemented inside the protocol selection screen, and not in the main UI panel.

See macOS and iOS implementations below.

macOS:
![macOS](https://user-images.githubusercontent.com/16647612/134323997-ebcea663-b7c3-4747-9399-eddb0c98caeb.png)
iOS:
![iOS](https://user-images.githubusercontent.com/16647612/134324129-90a3df1d-2f5f-4b0e-878f-ae0c211241e4.png)
